examples/flutter_gallery/lib/demo/material/icons_demo.dart

```diff
@@ -44,7 +44,34 @@ class IconsDemoState extends State<IconsDemo> {
     });
   }
 
-  Widget buildIconButton(double iconSize, IconData icon, bool enabled) {
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: const Text('Icons')
+      ),
+      body: new IconTheme(
+        data: new IconThemeData(color: iconColor),
+        child: new Padding(
+          padding: const EdgeInsets.all(24.0),
+          child: new Column(
+            children: <Widget>[
+              new _IconsDemoCard(handleIconButtonPress),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _IconsDemoCard extends StatelessWidget {
+
+  final VoidCallback handleIconButtonPress;
+
+  const _IconsDemoCard(this.handleIconButtonPress);
```

Comment:
always put the constructor first, that way we can tell at a glance if there's any constructors or if there's an implicit constructor instead.